deployments/charts/kuma/templates/pre-upgrade-install-missing-crds-job.yaml

```diff
@@ -115,12 +115,12 @@ spec:
       {{- end }}
       restartPolicy: OnFailure
       securityContext:
-      {{- toYaml (mergeOverwrite (dict "runAsNonRoot" true) .Values.hooks.podSecurityContext) | trim | nindent 8 }}
+      {{- toYaml .Values.hooks.podSecurityContext | trim | nindent 8 }}
       containers:
         - name: pre-upgrade-job
           image: {{ include "kubectl.formatImage" (dict "image" .Values.kubectl.image "root" $) | quote }}
           securityContext:
-          {{- toYaml (mergeOverwrite (dict "runAsUser" 65534 "readOnlyRootFilesystem" true) .Values.hooks.containerSecurityContext) | trim | nindent 12 }}
+          {{- toYaml (mergeOverwrite (dict "runAsUser" 65534) .Values.hooks.containerSecurityContext) | trim | nindent 12 }}
```

Comment:
so the user is still not an option?

Reply:
No, in fact the only solution to this would be to create a new value `kubectl.containerSecurityContext`

Reply:
On the other hand, it's very unlikely that it matters which non root user runs the `kubectl` image.

Reply:
We have to set it to something here because its user is `root` ATM. We could also rebuild the `kubectl` image (it's now set to use not root) , update the default image tag and leave `runAsUser` out here.

Reply:
Ho that's for the hook stuff. Yeah that's find then :)